lib/ui/painting.dart

```diff
@@ -847,6 +847,45 @@ abstract class Image extends NativeFieldWrapperClass2 {
 /// Callback signature for [decodeImageFromList].
 typedef void ImageDecoderCallback(Image result);
 
+/// Information for a single animation frame.
+///
+/// Obtain a FrameInfo with [Codec.getNextFrame].
+abstract class FrameInfo extends NativeFieldWrapperClass2 {
+  // The duration this frame should be shown.
+  int get durationMillis native "FrameInfo_durationMillis";
+
+  // The Image object for this frame.
+  Image get image native "FrameInfo_image";
+
+  /// Release the resources used by this object. The object is no longer usable
+  /// after this method is called.
+  void dispose() native "FrameInfo_dispose";
```

Comment:
Move this to the Codec class.  The Codec native peer registers a dispose method, and the FrameInfo native peer probably won't need one.

Reply:
done